tests/test_trello_api.py

```diff
@@ -0,0 +1,82 @@
+"""Integration tests for trello_api module"""
+
+import os
+import pytest
+import re
+from dotenv import find_dotenv, load_dotenv
+from todo_app.app import create_app
+from todo_app.trello_api import doing_list_name, done_list_name, todo_list_name
+from unittest.mock import Mock, patch
+
+TEST_BOARD_ID = '1'
+TEST_TODO_LIST_ID = '1'
+TEST_DOING_LIST_ID = '2'
+TEST_DONE_LIST_ID = '3'
+TEST_TODO_CARD_ID = '1'
+TEST_TODO_CARD_NAME = 'Task 1'
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = create_app()
+    with test_app.test_client() as client:
+        yield client
+
+@patch('todo_app.trello_api.requests.get')
+@patch('todo_app.trello_api.requests.post')
```

Comment:
Simpler and more robust version:

```suggestion
@patch('requests.get')
@patch('requests.post')
```